napari/layers/points/_points_utils.py

```diff
@@ -248,3 +249,23 @@ def fix_data_points(
             )
         ndim = data_ndim
     return points, ndim
+
+
+def coerce_symbols(array: np.ndarray) -> np.ndarray:
+    """
+    Parse an array of symbols and convert it to the correct strings.
+
+    Ensures that all strings are valid symbols and converts aliases.
+
+    Parameters
+    ----------
+    array : np.ndarray
+        Array of strings matching Symbol values.
+    """
+    # dtype has to be object, otherwise np.vectorize will cut it down to `U(N)`,
+    # where N is the biggest string currently in the array. This breaks if
```

Comment:
Incomplete sentence: `This breaks if...`

Finish the sentence or remove it.

Reply:
Nice catch; it was a leftover from when I didn't have a workaround for the np.vectorize probem.